src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Machine - truck started working");
```

Comment:
The message should indicate the specific type of `Machine` that is working. In this case, it should be 'Truck' instead of 'Machine - truck'. This makes the message more informative as per the checklist.